samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Web.config

```diff
@@ -1,131 +1,131 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
-  For more information on how to configure your ASP.NET application, please visit
-  https://go.microsoft.com/fwlink/?LinkId=301880
-  -->
-<configuration>
```

Comment:
If there are no important changes in the web.config, can we please exclude it from the PR?

Reply:
@laura-rodriguez The change to v3.6 is important. It's the only way I have found to compile the samples without errors. If I leave it at 2.0.1 and compile, VS does not generate the required Roslyn directories while building the sample.

Reply:
please ignore. Have removed vb line from bottom of web.config line as requested elsewhere.